src/Styleguide/Elements/Flex.tsx

```diff
@@ -1,19 +1,24 @@
 import styled from "styled-components"
 import {
   alignContent,
+  AlignContentProps,
   alignItems,
   AlignItemsProps,
   flexDirection,
   FlexDirectionProps,
   justifyContent,
   JustifyContentProps,
+  maxWidth,
+  maxHeight,
   space,
   SpaceProps,
   HeightProps,
   WidthProps,
   height,
   width,
   style,
+  MaxHeightProps,
+  MaxWidthProps,
```

Comment:
Can these be moved up near their associated functions?

Reply:
Sure thing.